onetimes.yml

```diff
@@ -209,6 +209,15 @@ events:
       Etherpad: https://input.osb-alliance.de/p/2022-10-scs-iam-workshop
       Coordinator: Felix Kronlage-Dammers <fkr[at]osb-alliance.com>
     location: "https://conf.scs.koeln:8443/SCS-Tech"
+  - summary: "SIG Monitoring - Audit Logging kick-off"
+    begin: 2022-11-21 11:00:00         # uses default timezone above
+    duration:
+      minutes: 60
+    description: |
+      SIG Monitoring Audit logging kick-off with STACKIT. Define which things are in scope ("how to get the right info from OpenStack", "which tools can be used how to audit log OpenStack") and then hope for a fruitful basis of future collaboration.
+      Etherpad: https://input.osb-alliance.de/p/2022-11-scs-monitoring-audit-logging
+      Coordinator: Jonas Schäfer <jonas.schaefer[at]cloudandheat.com>
+    location: tbd
```

Comment:
just use: https://conf.scs.koeln:8443/SCS-Tech

Reply:
apparently SIG-Community is at the same slot (thanks @frosty-geek). Then just .../SCS-SIG-Monitoring? :)